### ISSUE TITLE
In the case that the player goes straight to Averguard suddenly  and directly just after starting the game.

### ISSUE THREAD

**Issue by sujiniku** [clintbellanger/flare-game#577]
At the case of that the player has never met Martigan , and the case of never go into the mineshaft Longsword , 
and the case if  the player go to Avergurad suddenly , 
and the case if the player meet Guill at the entrance, 

At above case, for some reason Guill knows the work of the player, and Guill know that the player job is defeating the monsters, 
He appreciate the visit of the player.

Why can Guill know the player job?

In other words, I think that if player has never met Martigan until first encounter with Guill, it should be added a message to self introduce by player to Guill .

**Comment by dorkster** [comment]
I don't see the issue here. Guill's dialog is independent of the other NPCs and can stand on its own. Guill knows the player is suitable for fighting monsters by observing that they are carrying a weapon and are likely suited with some armor. In development, Guill was the only NPC for some time, so he, along with the Averguard quests, were designed to stand alone.

As a side note, I'd like to state that you shouldn't worry too much about the `alpha_demo` content, which is severely outdated. The primary flare-game content is the `empyrean_campaign` mod.
